docs/configuration/config-file-reference.md

```diff
@@ -416,6 +416,10 @@ cluster:
 # CLI flag: -experimental.alertmanager.enable-api
 [enable_api: <boolean> | default = false]
 
+# Maximum number of concurrent GET API requests before returning 503.
```

Comment:
Should it be a limit that we can override in tenant limit?

Reply:
Yes I think it makes sense that we can override this limit per tenant so we can adjust per tenant use case.

Reply:
I moved the configuration to tenant limit in the latest commit 🙏

Reply:
Moved back to static config because we don't create a new am instance when loading runtime config so we are not able to update this value at runtime.